app/views/building_map/index.html.erb

```diff
@@ -9,3 +9,51 @@
     polylines: BuildingMapHelper.leaflet_polylines(@route),
     markers: BuildingMapHelper.leaflet_markers(@route, @target))
 %>
+<style>
+.indoorLabel{
+  background: rgba(255, 255, 255, 0);
+  border:0;
+  border-radius:0px;
+  box-shadow: 0 0px 0px;
+}
```

Comment:
One quick suggestion from outside your team because I just saw your pull request: This should maybe move to `app/assets/stylesheets/building_map.scss` in order to have all CSS for that page in one place.

Reply:
Just copying it doesn't work (the style doesn't get applied). Is there anything else I should be aware of?

Reply:
Leaflet already defines styles for tooltips. Because these are loaded before `building_map.scss`, your styles get overwritten. Adding an `!important` at the end of each line should suffice to make it work again.

Reply:
Ah, thanks! I haven't done anything CSS-related for too long...